### ISSUE TITLE
Add aria-label to side nav in the left column of VAMC Staff profile page

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#16062]
## Description or Additional Context 
On Staff profile pages, there is a side nav in the left column. There is a `<nav>` element but is does not have a unique label to differentiate it from other navigations on the same page. Add an `aria-label` to the `<nav>` element.

Go to a Staff detail page, like [this one](https://www.va.gov/salem-health-care/staff-profiles/allen-r-moye/)
Inspect the sidenav
Notice that there is no aria-label
Also note that there is an empty `<ul>`

## Acceptance Criteria
- [ ] Add `aria-label="Secondary"` to the `<nav>` element.
- [ ] @laflannery has reviewed.

**Comment by laflannery** [comment]
@xiongjaneg I updated the AC to make it more specific with what the label should actually be

**Comment by randimays** [comment]
Validated successfully in production. Closing as complete

**Comment by laflannery** [comment]
This is all I've ever wanted in my life!
